webapp/src/main/java/org/apache/atlas/Atlas.java

```diff
@@ -150,7 +150,8 @@ public static void main(String[] args) throws Exception {
 
         server = EmbeddedServer.newServer(appHost, appPort, appPath, enableTLS);
         installLogBridge();
-
+        AtlasTopicCreator a = new AtlasTopicCreator();
+        a.createAtlasTopic(configuration, AtlasConstants.ATLAS_KAFKA_TAG_TOPIC);
         server.start();
```

Comment:
Can we do this topic creation from elsewhere? This can be here for now but I feel an infra related stuff should be moved elsewhere - probably Horizon or other places.